DOB

Tejas2805 opened the pull request:
@sreycodes please do see.

Tejas2805 committed:
Implemented DOB field for customer.Tests left.

Tejas2805 committed:
Implemented DOB field for customer.Tests left.

Tejas2805 committed:
Merge branch 'master' of https://github.com/Tejas2805/main

* 'master' of https://github.com/Tejas2805/main:
  Remove unused imports to please checkstyle
  Fix unused private fields to please Codacy
  Update the photo of nerrons(Shaoqian)
  Fix the errors caused by merging
  Fix the errors caused by merging
  Move the Ui image before the text
  Fix the typo in the link
  Update ReadMe doc
  Add use cases, new sequential diagrams and the new title to DG

Tejas2805 committed:
Merge branch 'master' into master

Tejas2805 committed:
Made main window bigger

Tejas2805 committed:
Merge branch 'resize-window'

Tejas2805 committed:
Merge branch 'master' of https://github.com/Tejas2805/main

ZhangCX10032 commented:
I am thinking that it might be better to name the variables as dateOfBirth instead of dob since dateOfBirth seems more straight forward for me.

Tejas2805 committed:
To implement DateOfBirth

Tejas2805 committed:
Added DateOfBirth completely.
Tests fo ID and DOB done.
Only SystemTests left. To be done when all Systems Tests work.

Tejas2805 committed:
Please Codacy

Tejas2805 committed:
Please Codacy

Tejas2805 committed:
Fixed codacy issues

Tejas2805 committed:
Fixed merge conflicts

Tejas2805 committed:
Corrected errors to please Travis

Tejas2805 committed:
Correct checkStyle errors

Tejas2805 committed:
Corrected errors

Tejas2805 committed:
Corrected errors

Tejas2805 committed:
Corrected errors.

Tejas2805 committed:
Error correction

Tejas2805 committed:
Corrected DOB.

Tejas2805 committed:
Change to please codacy

Tejas2805 committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-t12-1/main

* 'master' of https://github.com/cs2103-ay1819s2-t12-1/main: (34 commits)
  Revert back to original
  Fixed all system tests
  reverted image path in file
  Reverting file name in file
  Added back image file
  renamed file for consistency
  reverted file name for consistency
  Fixed checkstyle errors
  Chnaged message
  Added delete and edit commands
  Added important files
  Removed duplicate files
  Please checkstyle
  Fix SelectCommandSystemTest
  Fix HelpCommandSystemTest
  Fix FindCommandSystemTest
  Fix DeleteCommandSystemTest
  Fix ClearCommandSystemTest
  Fix AddCommandSystemTest
  Project built successfuly
  ...

# Conflicts:
#	src/main/java/seedu/hms/logic/commands/AddCustomerCommand.java
#	src/main/java/seedu/hms/logic/commands/EditCustomerCommand.java
#	src/main/java/seedu/hms/logic/parser/AddCustomerCommandParser.java
#	src/main/java/seedu/hms/logic/parser/EditCustomerCommandParser.java
#	src/main/java/seedu/hms/logic/parser/ParserUtil.java
#	src/main/java/seedu/hms/model/customer/Customer.java
#	src/main/java/seedu/hms/model/util/SampleDataUtil.java
#	src/main/java/seedu/hms/storage/JsonAdaptedCustomer.java
#	src/main/resources/view/CustomerListCard.fxml
#	src/test/data/JsonAddressBookStorageTest/invalidAndValidCustomerAddressBook.json
#	src/test/data/JsonAddressBookStorageTest/invalidCustomerAddressBook.json
#	src/test/data/JsonSerializableAddressBookTest/duplicateCustomerAddressBook.json
#	src/test/data/JsonSerializableAddressBookTest/invalidCustomerAddressBook.json
#	src/test/data/JsonSerializableAddressBookTest/typicalCustomersAddressBook.json
#	src/test/java/guitests/guihandles/CustomerCardHandle.java
#	src/test/java/seedu/address/testutil/TypicalCustomers.java
#	src/test/java/seedu/hms/logic/LogicManagerTest.java
#	src/test/java/seedu/hms/logic/commands/CommandTestUtil.java
#	src/test/java/seedu/hms/logic/commands/EditCustomerDescriptorTest.java
#	src/test/java/seedu/hms/logic/parser/AddCustomerCommandParserTest.java
#	src/test/java/seedu/hms/logic/parser/EditCustomerCommandParserTest.java
#	src/test/java/seedu/hms/logic/parser/ParserUtilTest.java
#	src/test/java/seedu/hms/model/customer/AddressTest.java
#	src/test/java/seedu/hms/model/customer/CustomerTest.java
#	src/test/java/seedu/hms/storage/JsonAdaptedCustomerTest.java
#	src/test/java/seedu/hms/testutil/CustomerBuilder.java
#	src/test/java/seedu/hms/testutil/CustomerUtil.java
#	src/test/java/seedu/hms/testutil/EditCustomerDescriptorBuilder.java
#	src/test/java/seedu/hms/ui/CustomerListPanelTest.java
#	src/test/java/systemtests/AddCommandSystemTest.java
#	src/test/java/systemtests/HotelManagementSystemSystemTest.java

Tejas2805 committed:
Correct for dob

Tejas2805 committed:
Corrected for Codacy

Tejas2805 committed:
Commit to match local and remotCommit to match local and remote
Commit to match local and remote
Commit to match local and remote

Tejas2805 committed:
Merge branch 'master' of https://github.com/Tejas2805/main

* 'master' of https://github.com/Tejas2805/main: (34 commits)
  Revert back to original
  Fixed all system tests
  reverted image path in file
  Reverting file name in file
  Added back image file
  renamed file for consistency
  reverted file name for consistency
  Fixed checkstyle errors
  Chnaged message
  Added delete and edit commands
  Added important files
  Removed duplicate files
  Please checkstyle
  Fix SelectCommandSystemTest
  Fix HelpCommandSystemTest
  Fix FindCommandSystemTest
  Fix DeleteCommandSystemTest
  Fix ClearCommandSystemTest
  Fix AddCommandSystemTest
  Project built successfuly
  ...

Tejas2805 committed:
Correcting stuff

Tejas2805 committed:
Correcting stuff.

Tejas2805 committed:
Correcting stuff.

Tejas2805 committed:
Still correcting.

Tejas2805 committed:
Correcting stuff

Tejas2805 committed:
Correct stuff

Tejas2805 committed:
Corrected

Tejas2805 committed:
Corrected.

Tejas2805 committed:
Corrected.

Tejas2805 committed:
Corrected errors from previous merges

Tejas2805 committed:
Correcting tests

Tejas2805 committed:
Corrected system tests

Tejas2805 committed:
Corrected for tests

Tejas2805 committed:
Corrected checkstyle errors

Tejas2805 committed:
Corrected checkStyle errors.

Tejas2805 committed:
DOB working.

Tejas2805 committed:
Corrected checkStyle errors.

Tejas2805 committed:
Corrected checkStyle error

Tejas2805 committed:
Correct errors.

Tejas2805 committed:
Corrected errors.

Tejas2805 committed:
Corrected checkStyle errors
